grails-app/services/org/pih/warehouse/inventory/StockMovementService.groovy

```diff
@@ -745,7 +749,7 @@ class StockMovementService {
         }
     }
 
-    def getStockMovementItems(String id, String stepNumber, String max, String offset) {
+    def getStockMovementItems(String id, String stepNumber, String max, String offset, Boolean createNewPicklist = true) {
```

Comment:
I'm generally against having that many arguments and we wanted to avoid that, but as a "make it work" solution this could be ok, because refactoring this method to receive e.g. a command object would be painful, wouldn't it?

Reply:
yes, having so many arguments can get messy, maybe we can agree on an approach where we pass `PaginationCommand` in these methods.
We can mimick the `Pageable` type from springboot where this object handles max, offset and sorting parameters